H2Codez/Common/H2EKCommon.cpp

```diff
@@ -408,6 +408,12 @@ void H2CommonPatches::Init()
 
 	set_tag_data_max_size(0x3000000);
 
+	if (conf.getBoolean("remove_data_size_limit", true))
+	{
+		set_tag_data_max_size(INT_MAX);
```

Comment:
idk dude INT_MAX might be too large, maybe INT_MAX - 5

Reply:
default to false

Reply:
and a setting to set make size would be best, dunno if we have a range limited setting option

Reply:
Idk if there are no issues with putting the size limit to its max there isn't really a point of putting a setting for how big you want it in (so far I haven't seen any issues when putting it to that)